<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>G3dwhDialog</class>
 <widget class="QDialog" name="G3dwhDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google 3D Warehouse</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QWebView" name="warehouseView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="downloadList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QProgressBar" name="downloadProgress">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="modelLabel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>Downloaded models</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
